parser.js

```diff
@@ -0,0 +1,113 @@
+import { RootNode, Node, TextNode } from './class.js';
+
+const removeLineBreaks = (str) => str.replace(/(\r\n|\n|\r)/gm, '');
+
+/* @params attributesToBeParsed : ['class="container"'] */
+/* @return attributes : [['class', 'container']] */
```

Comment:
Comment도 꼼꼼히 작성해주셨네요 :+1: 

참고로 JSDoc Comment는 `/**  */` 과 `@` 어노테이션으로 구성되어 있어요. 
아래와 같이 작성하시면 vscode 내에서 해당 함수에 마우스를 호버하면, JSDoc Comment가 표시된답니다 :smile: 

```suggestion
/**
  @params attributesToBeParsed : ['class="container"']
  @return attributes : [['class', 'container']]
*/
```

![image](https://user-images.githubusercontent.com/31029000/136951845-0ff2b313-bb15-4007-8b33-a1db0ae4f845.png)
![image](https://user-images.githubusercontent.com/31029000/136951886-15d1c05b-8456-44be-b7a0-f6b9ed1b3b3f.png)

Reply:
이 편리한 기능을...! 낫 놓고 기억자도 몰랐네요 🥲
마우스 hover 했을 때 정보 뜨는 거 너무 매력적이네요... 🥺 이 맛에 다들 타입 스크립트 쓰시는 건가요,,,? 

https://jsdoc.app/tags-param.html#overview
동동 덕분에 JS Doc 문서 다시 확인해보고 규칙에 맞게 수정해두었습니다! 
제 뇌 저편의 무의식에서 `params`, `return` 이라고 쓰라고 했는데 `param`, `returns`가 맞는 거였더라구요...! 제안해주신 내용 참고해서 커밋 따로 올렸습니당 :) 좋은 피드백 감사해용 > <

Reply:
📌 관련 커밋: e0d2dce

Reply:
타입까지 꼼꼼히 잘 기재해주셨네요 :+1: :+1: :+1: